nilearn/plotting/tests/test_img_plotting/test_plot_connectome.py

```diff
@@ -1,6 +1,7 @@
 """Tests for :func:`nilearn.plotting.plot_connectome`."""
 
 import os
+from pathlib import Path
```

Comment:
I suspect the import is not needed anymore: run precommit or ruff to clean this up.

Reply:
great ! thanks !